### PR TITLE
proto: add StoreState field and StoreIsTombstoneError field.

### PR DESCRIPTION
**Pull request by huachaohuang**
ref https://github.com/pingcap/pd/pull/298

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -250,7 +250,11 @@ message BootstrappedError {
...
 }
 
+message StoreIsTombstoneError {
+}
+
 message Error {
-    optional string message                 = 1;
-    optional BootstrappedError bootstrapped = 2;
+    optional string message                     = 1;
+    optional BootstrappedError     bootstrapped = 2;
```
Comment (ngaut) [inline proto/pdpb.proto]: Should we remove the Error suffix ?

Comment (huachaohuang) [inline proto/pdpb.proto]: I don't think so, this should be a `BootstrappedError` message, not a `Bootstrapped` message.

Comment (siddontang) [inline proto/pdpb.proto]: Maybe we can remove to BootstrappedFailed and StoreIsTombstone ?

Comment (huachaohuang) [inline proto/pdpb.proto]: I think is fine to add `Error` suffix to error message, just like request message named `IsBootstrappedRequest`.

**Comment by siddontang** [comment]
LGTM

**Comment by huachaohuang** [comment]
PTAL

**Comment by zhangjinpeng1987** [comment]
LGTM

**Pull request merged**